3rdParty/linenoise-ng/src/linenoise.cpp

```diff
@@ -483,7 +483,7 @@ class Utf32String {
   Utf32String& operator=(const Utf32String& that) {
     if (this != &that) {
       delete[] _data;
-      _data = new char32_t[that._length]();
+      _data = new char32_t[that._length + 1]();
```

Comment:
Why the extra Byte, and should it be initialized to 0 ?

Reply:
The compiler complained about a zero-length initialization here. The change ensures the warning will not occur.
The array is initialized with zeros because of the `()`.